libs/components/router/testing/src/href/href-harness.ts

```diff
@@ -0,0 +1,42 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyHrefHarnessFilters } from './href-harness-filters';
+
+export class SkyHrefHarness extends SkyComponentHarness {
+  static hostSelector = '[skyHref]';
```

Comment:
I'd go ahead and throw a `public` in front of this property to be more explicit.

Reply:
Added public.